docs/scalars.md

```diff
@@ -166,5 +243,8 @@ from thirdpartylib import json_serialize_money, json_deserialize_money
 money = ScalarType("Money")
 money.set_serializer(json_serialize_money)
 money.set_value_parser(json_deserialize_money)
-money.set_literal_parser(json_deserialize_money)
 ```
+
+> **Note:** the previous versions of this document also introduced the `literal_parser`. However in the light of `literal_parser` [reference documentation being incorrect](https://github.com/graphql/graphql-js/issues/2567) and the very usefulness of custom literal parsers [being discussed](https://github.com/graphql/graphql-js/issues/2657) we've decided to don't document it here anymore.
```

Comment:
```suggestion
> **Note:** the previous versions of this document also introduced the `literal_parser`. However in the light of `literal_parser` [reference documentation being incorrect](https://github.com/graphql/graphql-js/issues/2567) and the usefulness of custom literal parsers [being discussed](https://github.com/graphql/graphql-js/issues/2657) we've decided to no longer document it in this section.
```

Reply:
Not sure if "section" is right here. It may be taken as section within this document instead of entire document. I've also removed mention of literal parser from parsers section.